backend/yandex/yandex.go

```diff
@@ -57,6 +61,17 @@ func init() {
 			err := oauthutil.Config("yandex", name, m, oauthConfig)
 			if err != nil {
 				log.Fatalf("Failed to configure token: %v", err)
+				return
+			}
+			token, err := oauthutil.GetToken(name, m)
+			if err != nil {
+				fmt.Printf("Couldn't read OAuth token (this should never happen).")
+				return
+			}
+			token.TokenType = "OAuth"
```

Comment:
Yandex expects the OAuth token type to be "OAuth" however the token returned during  config is a standart "Bearer" token. Right know I rewrite the token during config to change it's type. The old implementation solved this problem by setting the Authorization header manually.